lib/metatron/templates/concerns/pod_producer.rb

```diff
@@ -31,6 +31,7 @@ def pod_producer_initialize
           @pod_annotations = {}
           @termination_grace_period_seconds = nil
           @tolerations = []
+          @volume_claim_templates = []
```

Comment:
I believe you meant to use this here:

```suggestion
          @persistent_volume_claims = []
```

Since I think you're trying to make the default for this an empty Array.

Reply:
oh woops!